_redirects

```diff
@@ -281,6 +297,7 @@ https://tutorial.getdbt.com/*	https://docs.getdbt.com/:splat	301!
 /reference/project-configs/modules-paths  /reference/project-configs/packages-install-path 302
 /docs/dbt-cloud/using-dbt-cloud/cloud-slack-notifications   /docs/dbt-cloud/using-dbt-cloud/cloud-notifications 302
 /reference/warehouse-profiles/presto-profile    /reference/profiles.yml 302
+/setting-up#generate-bigquery-credentials    /guides/getting-started/getting-set-up/setting-up-bigquery   302
```

Comment:
All old urls look to be working now besides this one. I wonder if it's because it includes `#generate-bigquery-credentials`. Want to try removing that to see if that fixes this redirect. All else looks g2g once this one is sorted out!

Reply:
Looks like removing that did the trick 👍